python/cudnn/frost/buffers.py

```
@@ -98,18 +98,80 @@ def dtype_name(buf) -> str:
+    1.68 us to fill vs 0.45 us to memmove the 72 bytes. Cached per (shape,
+    dtype, device) — a graph reuses a handful of layouts across its whole run.
+
+    This is the degenerate case of what the backend already does for kernel
+    arguments (``src/common/include/runtimeKernel.h``): a prefilled blob plus,
+    per mutable field, an (offset, uid, UpdateMethod) telling execute what to
+    write where. Here there is exactly one mutable field, ``data``, at a fixed
+    offset, and its update method is always POINTER — so the bookkeeping
+    collapses to a memmove and one assignment.
+    """
+
+    __slots__ = ("_template", "_shape_arr", "_nbytes")
```

Ruff (0.16.1)

[warning] `_DLPackProto.__slots__` is not sorted

Apply a natural sort to `_DLPackProto.__slots__`

(RUF023)